src/test/integration/api/retrieveUserUuid.test.js

```diff
@@ -0,0 +1,42 @@
+/**
+ * Created by SCMORETO on 19/04/2017.
+ */
+
+import retrieveUuid from "../../../main/api/retrieveUuid";
+import {UnexpectedErrorException, UnexpectedSuccessException, MappingSetupException} from "./testingErrors";
+import {raiseOrPassError, expectNotFoundOrRethrowError} from "./testingHelpers";
+
+
+describe('getUUID', () => {
+    it('should return a UUID for user DanC', () => {
+        return retrieveUuid.getUUID("dan.cotton@capgemini.com")
+                .then((result) => {
+                expect(result).toEqual(
+                    '26667c9fdcc603ee93b43fb3e780b07378695a86'
```

Comment:
these mocks should be updated. We need to differentiate between the result from the server-side and the client side. Otherwise the server sending without auth header will still pass.